scripts/android/create_android.sh

```diff
@@ -37,23 +40,26 @@ echo "Copying library files to Android app: ${PATH_TO_ANDROID_REPO}"
 
 PATH_TO_ANDROID_MAIN=$PATH_TO_ANDROID_REPO/app/src/main
 
+rm -fr $PATH_TO_ANDROID_MAIN/jniLibs
 mkdir $PATH_TO_ANDROID_MAIN/jniLibs
 mkdir $PATH_TO_ANDROID_MAIN/jniLibs/arm64
 mkdir $PATH_TO_ANDROID_MAIN/jniLibs/arm64-v8a
 mkdir $PATH_TO_ANDROID_MAIN/jniLibs/armeabi
 mkdir $PATH_TO_ANDROID_MAIN/jniLibs/x86_64
 mkdir $PATH_TO_ANDROID_MAIN/jniLibs/x86
 
-# debug
-ln -s $(pwd)/target/aarch64-linux-android/debug/libcoepi_core.so $PATH_TO_ANDROID_MAIN/jniLibs/arm64/libcoepi_core.so
-ln -s $(pwd)/target/aarch64-linux-android/debug/libcoepi_core.so $PATH_TO_ANDROID_MAIN/jniLibs/arm64-v8a/libcoepi_core.so
-ln -s $(pwd)/target/x86_64-linux-android/debug/libcoepi_core.so $PATH_TO_ANDROID_MAIN/jniLibs/x86_64/libcoepi_core.so
-ln -s $(pwd)/target/armv7-linux-androideabi/debug/libcoepi_core.so $PATH_TO_ANDROID_MAIN/jniLibs/armeabi/libcoepi_core.so
-ln -s $(pwd)/target/i686-linux-android/debug/libcoepi_core.so $PATH_TO_ANDROID_MAIN/jniLibs/x86/libcoepi_core.so
-
-# release
-# ln -s $(pwd)/target/aarch64-linux-android/release/libcoepi_core.so $PATH_TO_ANDROID_MAIN/jniLibs/arm64/libcoepi_core.so
-# ln -s $(pwd)/target/aarch64-linux-android/release/libcoepi_core.so $PATH_TO_ANDROID_MAIN/jniLibs/arm64-v8a/libcoepi_core.so
-# ln -s $(pwd)/target/x86_64-linux-android/release/libcoepi_core.so $PATH_TO_ANDROID_MAIN/jniLibs/x86_64/libcoepi_core.so
-# ln -s $(pwd)/target/armv7-linux-androideabi/release/libcoepi_core.so $PATH_TO_ANDROID_MAIN/jniLibs/armeabi/libcoepi_core.so
-# ln -s $(pwd)/target/i686-linux-android/release/libcoepi_core.so $PATH_TO_ANDROID_MAIN/jniLibs/x86/libcoepi_core.so
+if [[ $* == *--release* ]]; then
+    # release
+    cp $(pwd)/target/aarch64-linux-android/release/libcoepi_core.so $PATH_TO_ANDROID_MAIN/jniLibs/arm64/libcoepi_core.so
```

Comment:
Why are we copying now?

Reply:
Folks developing on Windows were reporting issues with symlinks failing, so changing them to a copy fixed it for them.